ui-components/src/global/forms.scss

```diff
@@ -1,4 +1,10 @@
 .field {
+  --bordered-border-width: 1px;
+  --bordered-border-radius: var(--bloom-s1_5);
+  --bordered-vertical-padding: var(--bloom-s4);
+  --bordered-leftward-padding: 2.875rem;
```

Comment:
Unique value since the current implementation renders the checkbox as part of label::before and this padding is set from the label text rather than the box